climada/engine/uncertainty_quantification/calc_base.py

```diff
@@ -65,10 +65,10 @@ def check_distr(self):
                 if input_param_name in distr_dict:
                     func = distr_dict[input_param_name]
                     x = np.linspace(func.ppf(0.01), func.ppf(0.99), 100)
-                    if not np.all(func.pdf(x) == input_param_func.pdf(x)):
+                    if not np.all(func.cdf(x) == input_param_func.cdf(x)):
```

Comment:
The pdf does not exist for discrete distribution, the cdf however is always defined.